corehq/apps/locations/migrations/0018_remove_related_locations.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.27 on 2020-02-12 22:56
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('locations', '0017_locationrelation_last_modified'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='locationrelation',
+            unique_together=set([]),
+        ),
+        migrations.RemoveField(
+            model_name='locationrelation',
+            name='location_a',
+        ),
+        migrations.RemoveField(
+            model_name='locationrelation',
+            name='location_b',
+        ),
+        migrations.AlterModelManagers(
+            name='sqllocation',
+            managers=[
+            ],
+        ),
+        migrations.DeleteModel(
+            name='LocationRelation',
+        ),
+    ]
```

Comment:
what does the sql output for this look like?

Reply:
```
BEGIN;
--
-- Alter unique_together for locationrelation (0 constraint(s))
--
ALTER TABLE "locations_locationrelation" DROP CONSTRAINT "locations_locationrelati_location_a_id_location_b_e6e38d65_uniq";
--
-- Remove field location_a from locationrelation
--
SET CONSTRAINTS "locations_locationre_location_a_id_3a57268d_fk_locations" IMMEDIATE; ALTER TABLE "locations_locationrelation" DROP CONSTRAINT "locations_locationre_location_a_id_3a57268d_fk_locations";
ALTER TABLE "locations_locationrelation" DROP COLUMN "location_a_id" CASCADE;
--
-- Remove field location_b from locationrelation
--
SET CONSTRAINTS "locations_locationre_location_b_id_d92e664d_fk_locations" IMMEDIATE; ALTER TABLE "locations_locationrelation" DROP CONSTRAINT "locations_locationre_location_b_id_d92e664d_fk_locations";
ALTER TABLE "locations_locationrelation" DROP COLUMN "location_b_id" CASCADE;
--
-- Change managers on sqllocation
--
--
-- Delete model LocationRelation
--
DROP TABLE "locations_locationrelation" CASCADE;
COMMIT;
```

Reply:
Realized there are conflicting migrations, added 59c68cd4b0c to delete the old one and re-run `makemigrations`. Now it's just `DROP TABLE "locations_locationrelation" CASCADE;`